spec/lib/whitehall_importer/create_revision_spec.rb

```diff
@@ -151,4 +193,19 @@
         .to eq(world_location["content_id"])
     end
   end
+
+  def create_image(filename)
+    image_url = "https://assets.publishing.service.gov.uk/government/uploads/#{filename}"
+    @binary_image ||= file_fixture("960x640.jpg").read
+    stub_request(:get, image_url).to_return(status: 200, body: @binary_image)
+    {
+      "id": 194072,
+      "alt_text": "Alt text for image",
+      "caption": "This is a caption",
+      "created_at": "2018-11-30T05:08:56.000+00:00",
+      "updated_at": "2018-11-30T05:19:53.000+00:00",
+      "variants": {},
+      "url": image_url,
+    }.with_indifferent_access
```

Comment:
I feel that this would be more consistent with the recent code if it was built using a factory bot factory.

That factory could also set up the webmock stubbing for the file.

Reply:
We also could maybe use the block argument of `to_return` so it only reads the file if the mock is called rather than needing to call it each time the mock is set up (I'm assuming this is why we've set up the instance variable)

Reply:
My first instinct was to try to make a factory for it, but as it is representing JSON from the Whitehall import rather than representing an instance of a model in Content Publisher, it didn't feel appropriate.

I'll take another look at factories now I know it's a sound idea!